### ISSUE TITLE
Card on Hover

### ISSUE THREAD

**Issue by abramjstamper** [octoshrimpy/bulma-o-steps#12]
Is there any support for adding a component like a card or box on hover?

**Comment by octoshrimpy** [comment]
@abramjstamper could you elaborate a bit more?

**Comment by abramjstamper** [comment]
Of course. I was working with this in React so the term component made sense to use. I tried to make a JSFiddle of what I was asking about to help clarify. My example is missing the `onHover` because I hardcoded a `card` as the `step-content`'s child. Is there any additional classes to show/hide a `card` or `step-content` when a user click/hovers over a `step-segment`?


https://jsfiddle.net/pfb1kvtd/6/

**Comment by octoshrimpy** [comment]
that can be done for sure. I'll need a bit of time to get it all done, but that can be done.

**Comment by octoshrimpy** [comment]
now under work.

**Comment by nikhilkadyan** [comment]
any update on this ?
